src/sentry_plugins/vsts/client.py

```diff
@@ -13,12 +13,12 @@
 
 
 class VstsClient(AuthApiClient):
-    api_version = '3.0'
+    api_version = '4.1'
 
     def request(self, method, path, data=None, params=None):
         headers = {
             'Accept': 'application/json; api-version={}'.format(self.api_version),
-            'Content-Type': 'application/json-patch+json',
+            'Content-Type': 'application/json',
```

Comment:
:-|

Reply:
Are you sure this doesn’t break things? It was required before.

Reply:
do you remember why? it was causing the commit fetching requests to 500 for me 😬

Reply:
Their API required it for PATCH requests. If you prove it still works with any call that does PATCH within this patch then that'd satisfy my concern. It's possible the new version of the API is fine now without it.